### DISCUSSION TITLE
Q7 Max Version and HowTo?

### DISCUSSION THREAD

**Discussion by DSDV** [Hypfer/Valetudo#1827] [Q&A / Support] [answered]
Hi fellow community <3, 

so i got a Q7 Max today which i ordered as an Upgrade for my S5e Max. (supposedly better suction etc). 
where do i see if it is Version ver 0866, 11/2022 or ver 0906, 12/2022?

with kind regards

//EDIT: i found a few tutorials :D
https://valetudo.cloud/pages/installation/roborock.html#fel / https://builder.dontvacuum.me/howtos/nand/fel/_howto.html

**Comment by Hypfer** [comment, marked as answer]
0866 and 0906 refer to firmware versions.
You don't need to match the firmware version for the rooted image with the version that is currently installed.

**Reply by DSDV** [reply]
ohh ok thanks for clearing that up (: 
so just take the latest ? :D

**Comment by DSDV** [comment]
![temp](https://github.com/Hypfer/Valetudo/assets/5504857/874e9a41-bbc4-4ba7-b690-20bf010f280a)

anyone has a clue where i can find GND ? 
i only see TPA 17

**Reply by DSDV** [reply]
i was ready to gamble and took the lower patch of the 3 silver rectangles and it worked :D

**Comment by DSDV** [comment]
> If it’s a used robot, you might not see an AP. In that case, press and hold the two outer buttons until you see the Wi-Fi LED change. It should then spawn the AP again.

reading is an art (: - this block should be in yellow for stupid folks like me :D

**Comment by elba96** [comment]
Sorry to stir up such an old discussion but I have troubles with my Q7 Max as well.

I can't even disassemble my unit, all disassembly videos I found on YT are for the S7 model, and now I can't even reach the motherboard. @DSDV would you be willing to help me with this?

**Reply by DSDV** [reply]
Hi (: 
i had a S5 Max before and the disassembly process somewhat the same. 
did you take a look at the 1st of my two posted links? (valetudo)

For disassembly i recommend making a bit of space on the floor and place all parts and screws in a explosion diagram way so you can get everything together afterwards ;)

if you have specific question i may be able to help more

**Reply by elba96** [reply]
Hey!

Thanks for the reply! I completely forgot about this issue, but I figured it out eventually.
I looked at your resources, but my problem was that I couldn't take off the top cover of the robot and so I had no access to the mainboard. 
I somehow overlooked 1 screw that was hidden under a sticker (see the images), after that I just had to apply enough force to take off the top cover of the Robot and finally reached the mainboard.

![roborock](https://github.com/Hypfer/Valetudo/assets/58250156/f105d6f2-6e2c-4d1d-bf87-c39ab6f21ba6)


Now I'm running Valetudo without issues!

**Reply by DSDV** [reply]
good to hear :D
and have fun with a nice privacy minded cleaner <3
